src/engine/enginebuffer.cpp

```diff
@@ -416,7 +416,7 @@ void EngineBuffer::requestSyncMode(SyncMode mode) {
 }
 
 void EngineBuffer::requestClonePosition(EngineChannel* pChannel) {
-    m_pChannelToCloneFrom.store(pChannel);
+    atomicStoreRelaxed(m_pChannelToCloneFrom, pChannel);
```

Comment:
@holzhaus
Could it be this change is the cause for https://bugs.launchpad.net/mixxx/+bug/1860808 ?
"Deck Clone Begins Playback at Cue point"

Reply:
I don't think so. `atomicStoreRelaxed` just calls `m_pChannelToCloneFrom.store(pChannel)`: https://github.com/mixxxdj/mixxx/pull/2423/files#diff-5874728ca362c290855d2db35041fac3R159-R171

Reply:
Howerver, this is the cause:
when I revert it to `m_pChannelToCloneFrom.store(pChannel)` cloning works as before.
Seems the version switch in [`atomicStoreRelaxed`](https://github.com/mixxxdj/mixxx/pull/2423/files#diff-5874728ca362c290855d2db35041fac3R174) isn't working or something..
I also notice that the Play button on the clone target deck now always lights up again, which it didn't with `atomicStoreRelaxed`

Reply:
Why Qt version are you using?

Reply:
Qt 5.9.5

Reply:
Can you check if https://github.com/Holzhaus/mixxx/commit/cddd3e3fda507424b1d51e63cab6c1cd6aaf44b2 fixes the issue?
You can apply it directly on your branch:

```bash
$ wget https://github.com/Holzhaus/mixxx/commit/cddd3e3fda507424b1d51e63cab6c1cd6aaf44b2.patch
$ git am < cddd3e3fda507424b1d51e63cab6c1cd6aaf44b2.patch
```

Reply:
Ups, .. why we have not noticed this before. We should include this in any case. Please don't use non const references. Use pointer semantic in the store case.